sogs/static/output.css

```diff
@@ -0,0 +1,3 @@
var(--tw-grayscale) var(--tw-hue-rotate) var(--tw-invert) var(--tw-saturate) var(--tw-sepia) var(--tw-drop-shadow)}.transition{transition-duration:.15s;transition-property:color,background-color,border-color,fill,stroke,opacity,box-shadow,transform,filter,-webkit-text-decoration-color,-webkit-backdrop-filter;transition-property:color,background-color,border-color,text-decoration-color,fill,stroke,opacity,box-shadow,transform,filter,backdrop-filter;transition-property:color,background-color,border-color,text-decoration-color,fill,stroke,opacity,box-shadow,transform,filter,backdrop-filter,-webkit-text-decoration-color,-webkit-backdrop-filter;transition-timing-function:cubic-bezier(.4,0,.2,1)}.duration-500{transition-duration:.5s}.hover\:bg-black:hover{--tw-bg-opacity:1;background-color:rgb(0 0 0/var(--tw-bg-opacity))}.hover\:text-white:hover{--tw-text-opacity:1;color:rgb(255 255 255/var(--tw-text-opacity))}.dark .dark\:border-lightGray{--tw-border-opacity:1;border-color:rgb(88 88 88/var(--tw-border-opacity))}.dark .dark\:border-sessionGreen{--tw-border-opacity:1;border-color:rgb(0 233 123/var(--tw-border-opacity))}.dark .dark\:bg-lightGray{--tw-bg-opacity:1;background-color:rgb(88 88 88/var(--tw-bg-opacity))}.dark .dark\:bg-otherGray{--tw-bg-opacity:1;background-color:rgb(39 39 38/var(--tw-bg-opacity))}.dark .dark\:bg-sessionGreen{--tw-bg-opacity:1;background-color:rgb(0 233 123/var(--tw-bg-opacity))}.dark .dark\:text-white{--tw-text-opacity:1;color:rgb(255 255 255/var(--tw-text-opacity))}.dark .dark\:text-sessionGreen{--tw-text-opacity:1;color:rgb(0 233 123/var(--tw-text-opacity))}.dark .dark\:text-black{--tw-text-opacity:1;color:rgb(0 0 0/var(--tw-text-opacity))}.dark .dark\:decoration-white{-webkit-text-decoration-color:#fff;text-decoration-color:#fff}.dark .dark\:hover\:bg-sessionGreen:hover{--tw-bg-opacity:1;background-color:rgb(0 233 123/var(--tw-bg-opacity))}.dark .dark\:hover\:text-black:hover{--tw-text-opacity:1;color:rgb(0 0 0/var(--tw-text-opacity))}@media (min-width:365px){.xs\:text-4xl{font-size:2.25rem;line-height:2.5rem}.xs\:text-xl{font-size:1.25rem;line-height:1.75rem}.xs\:text-3xl{font-size:1.875rem;line-height:2.25rem}.xs\:text-lg{font-size:1.125rem;line-height:1.75rem}}@media (min-width:640px){.sm\:h-20{height:5rem}.sm\:grid-cols-3{grid-template-columns:repeat(3,minmax(0,1fr))}.sm\:flex-row{flex-direction:row}.sm\:justify-between{justify-content:space-between}.sm\:px-5{padding-left:1.25rem;padding-right:1.25rem}.sm\:pl-0{padding-left:0}.sm\:pt-2{padding-top:.5rem}.sm\:text-5xl{font-size:3rem;line-height:1}.sm\:text-2xl{font-size:1.5rem;line-height:2rem}.sm\:text-4xl{font-size:2.25rem;line-height:2.5rem}.sm\:text-xl{font-size:1.25rem;line-height:1.75rem}.sm\:text-base{font-size:1rem;line-height:1.5rem}}@media (min-width:768px){.md\:h-28{height:7rem}}@media (min-width:1024px){.lg\:grid-cols-4{grid-template-columns:repeat(4,minmax(0,1fr))}}@media (min-width:1536px){.\32xl\:grid-cols-6{grid-template-columns:repeat(6,minmax(0,1fr))}}
```

Comment:
I think this file is generated right?
If yes, it should be removed from the git and added to .gitignore. It's not supposed to be there if it's generated